loadtest/azuredeploy.parameters.loadtest.defaults.json

```diff
@@ -11,6 +11,7 @@
         "fileServerDiskCount":         { "value": 2 },
         "fileServerDiskSize":          { "value": 128 },
         "redisDeploySwitch":           { "value": false },
-        "sshPublicKey":                { "value": "__SSH_PUB_KEY__" }
+        "sshPublicKey":                { "value": "__SSH_PUB_KEY__" },
+        "azureSearchSwitch":	       { "value": true }
```

Comment:
We don't want Azure Search to be turned on in load testing, so please change this to false again. Better yet, we can depend on the default false value, so just remove this line.